FordLabs/PeopleMover
# Fix Husky/guet

## Issue
Connects A/N

## What was done
- [x] Moved `Husky` to the root folder so it will play well with `Guet`
- [x] Ensured husky runs eslint and ui unit tests on commit
- [x] Moved `Jest` config into it's own file (`ui/.jest.config.js`)
- [x] Moved remaining eslint config that was in the package.json into it's own file (`ui/.eslintrc.js`)
- [x] Deleted relic issue template (`.github/ISSUE_TEMPLATE/peoplemover-issue.md`)
- [x] Moved scripts to the top of the file in `ui/package.json` so that they are more easily found

## How to test
1. In the UI folder, run `npm install` to remove husky from the node_modules folder
2. In the root folder, run 
```
rm -rf .git/hooks
npm install
npm rebuild
guet start -a
```
3. Ensure a couple people are set in guet then make a dummy commit in the ui folder and or the root folder and ensure eslint AND ui unit tests run on commit.
